gamemodes/terrortown/gamemode/client/cl_status.lua

```diff
@@ -8,14 +8,19 @@ STATUS.active = {}
 
 ---
 -- Registers a @{STATUS}
--- @param number id The index of the new status
+-- @param number id the index of the new status
```

Comment:
btw. the previous one was right ;) everything after the param name is the string arg for the docu ^^

Reply:
You're right. My bad